src/tespy/components/heat_exchangers.py

```diff
@@ -2885,6 +2942,34 @@ def ttd_u_func(self):
         T_o2 = T_mix_ph(o2, T0=self.outl[1].T.val_SI)
         return self.ttd_u.val - T_bp_p(i1) + T_o2
 
+    dissipative = True
+    def exergy_balance(self, Tamb_val_SI, bus):
+        r"""
+        Calculate exergy balance of a condenser.
+
+        Parameters
+        ----------
+        Tamb_val_SI : float
+            Ambient temperature in K.
+        bus  : tespy.connections.bus
+            Energy flows in network. Used to calculate product exergy
+            or fuel exergy of turbines and pumps.
+            
+        Note
+        ----
+            .. math::
+
+                E_{P} = n/a
+                E_{F} = \dot{m}_{in} \cdot (e_{ph,in} - e_{ph,out})
+        """
+        if self.dissipative == True:
+            self.E_P = 'n/a'
+        if self.dissipative == False:
+            self.E_P = self.inl[1].m.val_SI * (self.outl[1].ex_physical
```

Comment:
use if ... else ... Where is self.dissipative defined? And how to change it?

Reply:
Ah yes, if.. else.. is better.
self. dissipative is defined right above the method exergy_balance 
In the user's code it can be changed by condensername.dissipative = False
Then, the condenser is treated as an ordinary heat exchanger and not as a dissipative component.